README.md

```diff
@@ -1,2 +1,372 @@
-# play-json-ops
-Common Play Json implicits and helper methods
+<a href='https://travis-ci.org/jeffmay/play-json-ops'>
+  <img src='https://travis-ci.org/jeffmay/play-json-ops.svg' alt='Build Status' />
+</a>
+<a href='https://coveralls.io/github/jeffmay/play-json-ops?branch=master'>
+  <img src='https://coveralls.io/repos/jeffmay/play-json-ops/badge.svg?branch=master&service=github' alt='Coverage Status' />
+</a>
+
+# Play Json Ops
+
+Augments the [Play Json library](https://www.playframework.com/documentation/2.5.x/ScalaJson) with some helpful
+implicits and tools for:
+
+- Creating formats for traits and abstract classes
+- Safely printing error messages with redacted sensitive data using implicit transformations
+- Formats for all tuples as JsArray
+- ScalaCheck generators for JsValue, JsArray, and JsObject
+- Formats for scala.concurrent.Duration
+- UTCFormats for org.joda.time.DateTime
+- Compile-time Json.oformat and Json.owrites macros (Play 2.3 only)
+
+<h1>Versions</h1>
+
+<h2>2.X Branch</h2>
+<table>
```

Comment:
You know you can write tables in markdown, right?

Reply:
It's fine, just letting you know, doesn't need to be a fast follow, sounds more like an enhancement that will come when someone gets fed up modifying the HTML table ;p